### ISSUE TITLE
ran into issue with the following command sls s3deploy

### ISSUE THREAD

**Issue by jgardezi** [matt-filion/serverless-external-s3-event#2]
Hi,

First `npm install serverless-external-s3-event` command does not work I have to use `npm install https://github.com/matt-filion/serverless-external-s3-event`. But its a seperate issue https://github.com/matt-filion/serverless-external-s3-event/issues/1.

When I executed command `sls s3deploy`. I am getting the following error and `sls deploy` command worked fine.

>> sls s3deploy

`
  Type Error ---------------------------------------------
 
     Cannot read property 'sdk' of undefined
 
     For debugging logs, run again after setting SLS_DEBUG env var.
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
 
     Please report this error. We think it might be a bug.

  Your Environment Information -----------------------------
     OS:                 darwin
     Node Version:       4.3.2
     Serverless Version: 1.2.1
`

Can someone help on this one?

Kind regards,
Javed Gardezi

**Comment by matt-filion** [comment]
Thank you for your feedback, I apologize I only noticed this comment. I don't spend much time on GitHub.

For #1 you are correct, this has not been publish to npmjs the central NPM repository. I was not satisfied with the state of the plugin (and some of the serverless capabilities to support the plugin) so I didn't want to announce it.

For #2, this is likely an incompatibility with the latest version of serverless. I wrote it against 1.0 beta. I'll put some effort towards both of these in the next coming weeks and let you know when everything works.

Matt Filion
